### PR TITLE
feat: Multiple categories per application

### PR DESCRIPTION
**Pull request by dievazqu**
### Description

Adding multiple categories per application.
To maintain backwards compatibility, I left the `categoryId` field in the json, but the newest version will use the array of `categories`.
The categories were assigned following [this](https://docs.google.com/document/d/1uWQFc98ENwFA2v88mq4IHaicXopmqGwKyKCpmTRWO0o)

Fixes RET-583

### CODE REVIEW

**Review by kathaypacific — APPROVED** [review]
LGTM! just a small comment about the translations that we should fix before merging :)

File: docs/adding-new-dapps.md [hunk 1]
```diff
@@ -17,7 +17,8 @@ There are some fields that need to be translated into different languages, and t
...
 | `name`            | The name of the Dapp.                                                                                                                                                                                                                                     |
 | `id`              | A unique identifier for the Dapp. The current pattern is to use a lower-kebab-case string.                                                                                                                                                                |
-| `categoryId`      | The category id under which the Dapp should show up.                                                                                                                                                                                                      |
+| `categoryId`      | The category id under which the Dapp should show up. This will be deprecated soon, it will be replaced by `categories`.                                                                                                                                   |
```
Comment (jeanregisser) [inline docs/adding-new-dapps.md:20]: I propose we directly remove `categoryId` here and update the Cloud Function to return the right shape based on the Valora version.
The first one would be used for existing clients.

**Review by jeanregisser — COMMENTED** [review]
Thanks! 👍

File: locales/de.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "categories": {
     "exchanges": "Tauschen",
-    "lend": "Leihen, verleihen und verdienen",
+    "lend-borrow-earn": "Leihen, verleihen und verdienen",
```
[the remark this answers is not in the document]
Comment (dievazqu) [reply locales/de.json:4]: Oh! You are right! I hit `search and replace` without too much thought!

File: docs/adding-new-dapps.md (hunk 1 above)
Comment (dievazqu) [reply docs/adding-new-dapps.md:20]: Yes, but what do we do with the apps that are currently in `Lend, Borrow & Earn`?
We have to split that category, and I didn't want to change the current behavior.

**Pull request merged**